W-15898668-check-logs-duke (#340)
mulesoft/docs-code-builder: modules/ROOT/pages/troubleshoot-missing-features.adoc

Some Anypoint Code Builder features can be disabled or hidden based on settings in the IDE. 

> For any features that are not working as expected, first xref:troubleshooting.adoc#check-logs[check project logs].

Author: ```suggestion
For any features that are not working as expected, first xref:troubleshooting.adoc#check-logs[check for problems and issues] .
```

> For any features that are not working as expected, first xref:troubleshooting.adoc#check-logs[check for problems and issues].

Reviewer: ```suggestion
For any features that aren't working as expected, first xref:troubleshooting.adoc#check-logs[check for problems and issues].
```
I believe our current guidance says to use contractions. But if I'm wrong, please disregard!

> For any features that are not working as expected, first xref:troubleshooting.adoc#check-logs[check for problems and issues] .

Author: ```suggestion
For any features that are not working as expected, first xref:troubleshooting.adoc#check-logs[check for problems and issues].
```
